packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -103,11 +109,11 @@ export class HttpServer extends RunnableModule {
       };
 
     this.app.use(
-      '/health',
+      path.join(versionPath, 'health'),
```

Comment:
Maybe instead of duplicating `path.join(versionPath, ...)` everywhere we can do sth like `this.app = express().use(versionPath)` and then mount (use) all subpaths on that.